admin/package.json

```diff
@@ -6,7 +6,7 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-scripts": "3.4.3",
-    "styled-components": "^5.2.0",
+    "styled-components": "^5.1.1",
```

Comment:
Any reason for decreasing version?

Reply:
Bacause 5.2.0 failed tests with no reason why.

Reply:
SC 5.2.0 isn't working correctly in ReactDOM and blocking updating packages on the Arch.